lib/fedex.rb

```diff
@@ -51,5 +51,12 @@
 module Fedex
   require 'fedex/version'
   #Exceptions: Fedex::RateError
-  class RateError < StandardError; end
+  class RateError < StandardError
+    attr_accessor :code
+
+    def initialize(msg = nil, code: nil)
```

Comment:
`kwargs` appear in ruby 2.0, are the maintainers OK with killing support for lower version?